src/pages/settings/Wallet/ExpensifyCardPage.js

```diff
@@ -74,58 +81,89 @@ function ExpensifyCardPage({
                         onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_WALLET)}
                     />
                     <ScrollView contentContainerStyle={safeAreaPaddingBottomStyle}>
-                        <View style={[styles.flex1, styles.mb4, styles.mt4]}>
+                        <View style={[styles.flex1, styles.mb9, styles.mt9]}>
                             <CardPreview />
                         </View>
 
-                        <MenuItemWithTopDescription
-                            description={translate('cardPage.availableSpend')}
-                            title={formattedAvailableSpendAmount}
-                            interactive={false}
-                            titleStyle={styles.newKansasLarge}
-                        />
-                        {!_.isEmpty(virtualCard) && (
+                        {isDetectedDomainFraud && (
```

Comment:
Let's use the ` Boolean ? () : null` pattern instead of `&&` for these.

Reply:
applied